ui/ui-components/utils/apiData.ts

```diff
@@ -343,12 +341,6 @@ export namespace Actions {
   export type DestinationProfilePreview = AsyncReturnType<
     typeof DestinationProfilePreview.prototype.runWithinTransaction
   >;
-  export type DestinationTrackGroup = AsyncReturnType<
-    typeof DestinationTrackGroup.prototype.runWithinTransaction
-  >;
-  export type DestinationUnTrackGroup = AsyncReturnType<
-    typeof DestinationUnTrackGroup.prototype.runWithinTransaction
-  >;
```

Comment:
Glad you found these!  This file is how the UI's know the TS Types of the Action's responses.